### ISSUE TITLE
Lazy/partial large zip-reading 

### ISSUE THREAD

**Issue by aleksas** [Stuk/jszip#576]
Can Rang'ed XMLHttpRequest (as indicated [here](https://stackoverflow.com/a/15561674/1433554)) be used to read parts of archive not to load it whole. I.e. read big zip header only, then on demand read requested archived file chunks.

**Comment by pwolf451** [comment]
Hi Aleksas,
Did you manage to solve the problem ? I am also interested by this issue.

Thank you.

**Comment by aleksas** [comment]
No, but it's not a pressing issue for me. Wasn't investigating any further.

**Comment by PhilippWendler** [comment]
This is indeed possible in principle, because the [zip.js library](https://github.com/gildas-lormeau/zip.js) also does it. I would also love to see support for this in JSZip. But this is currently not supported and a duplicate of #290.
